lib/galaxy_test/api/test_workflows.py

```diff
@@ -1352,6 +1367,15 @@ def test_workflow_invocation_report_custom(self):
             assert "## Workflow Inputs" in markdown_content
             assert "## About This Report" in markdown_content
 
+    @skip_without_tool("cat1")
+    def test_export_invocation_bco(self):
+        with self.dataset_populator.test_history() as history_id:
+            summary = self._run_jobs(WORKFLOW_SIMPLE, test_data={"input1": "hello world"}, history_id=history_id)
+            invocation_id = summary.invocation_id
+            bco = self._get("invocations/%s/export_bco" % invocation_id).json()
+            self.assertEqual(bco['spec_version'], 'https://w3id.org/biocompute/1.4.0/')
```

Comment:
@HadleyKing is there any tool/library that validates the BCO?

Reply:
Yes. both of these were put together by @corburn
This is a stand alone python
- https://github.com/biocompute-objects/BCO_Specification/blob/1.4.0/schemas/validate.py
this is a Github action
- https://github.com/corburn/actions

Reply:
@HadleyKing Yes. Thank you and @nsoranzo for getting this working!